CHARTER.md

```diff
@@ -0,0 +1,108 @@
+- Establishment of and delegation to working groups or teams
+- Mediating technical conflicts
+
+It is also responsible for establishing a Code of Conduct Committee suitable for mediating non-technical conflicts.
+In any period where such a committee is not yet formed, the TSC must assume temporary responsibility of mediating such conflicts in addition to responsibilities enumerated above.
+
+In joining the TSC, members commit to communicate on a regular basis and respond to issues raised by the TSC in a timely manner. If they are no longer able or willing to make such a commitment, they should discuss this with the TSC or a TSC Chair.
+
+### 4.1 Project Operations & Management
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+#### Decision-making
+
+The TSC follows a consensus-seeking decision making model in which joint decisions are agreed upon by all members whenever possible.
+In some situations, a vote may be preferable, however a formal vote is expected to be an infrequent occurrence invoked only when consensus cannot be reached after multiple attempts to reach TSC consensus.
+TSC discussions and decisions are to be assumed public by default, unless otherwise decided upon on a case-by-case basis by the TSC.
+Precise criteria for which decisions are to be left private are left to the TSC's discretion and assumed to include security-related reports or discussions with a third-party who wishes their interactions to remain private, such as when they concern yet unpublished case studies or partnerships which are not yet ready for announcement.
+
+The TSC is expected to agree upon and maintain specific process and procedure which facilitate labelling or communicating which decisions have been formally decided upon by the TSC.
+This process should include a mechanism for identifying ongoing TSC discussions and decision-making.
+When concluded, decisions should include reasoning and/or justification of the TSC, as well as indication of the consensus of the committee, or lack thereof in the case of a voted decision as indicated above.
+Private decisions should also be documented in a private location accessible to parties expected to have access to them.
+
+## Code of Conduct Findings & Violations
+
+Code of Conduct incidents must be reported to the TSC by the Code of Conduct committee.
+Reports must remain anonymous, as per the Code of Conduct, but should be documented in a manner similar to private TSC decisions as mentioned.
+
+## 5: Definitions
+
+TSC: The JSON Schema Technical Steering Committee, delegated technical leadership for the JSON Schema project by the OpenJS Foundation.
+
+---
+
+This work is licensed under a [Creative Commons Attribution-ShareAlike 2.0 UK: England & Wales License](https://creativecommons.org/licenses/by-sa/2.0/uk/).
```

Comment:
I think you need to add a copyright notice if you're adding a specific license. Also, this license isn't one of the approved licenses. I'm assuming that this is because part of the text was originally pulled from elsewhere. Ironically, this isn't an approved license for documents, so it might be worth checking whether there remains actual content from the original and if an acknowledgement of inspiration (and a CC-BY 4) isn't an acceptable option in that case.

Reply:
Good call. I removed/migrated the "This work is a derivative of..." text, because I didn't feel it was any longer, so this applies.

Do we need to/should we have any licence/copyright notice to this?
If it might be useful to others, I would want it to be copyable.

How do we proceed? =]

Reply:
Updated as discussed. Thanks.